companies/hagengrote.json

```
@@ -0,0 +1,20 @@
+{
+    "slug": "hagengrote",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Hagen Grote GmbH",
+    "address": "Grüner Dyk 36\n47803 Krefeld\nDeutschland",
```

If the privacy policy lists the address of an external data protection officer, we use that.

```suggestion
    "address": "c/o ZB Datenschutz und -sicherheit GmbH & Co. KG\nHansestraße A1/79\n51149 Köln\nDeutschland",
```